roles/apt/templates/30apt-proxy.j2

```diff
@@ -1,2 +1,2 @@
 // {{ ansible_managed }}
-Acquire::http::Proxy "{{ apt_proxy }}";
+Acquire::http::Proxy "http://{{ inventory_hostname }}:8000/";
```

Comment:
What I'm getting at is, `inventory_hostname` is the local machine's hostname. So every machine will be pointing *at itself* as an APT proxy. That can't be right.